celesta-sql/src/main/java/ru/curs/celesta/dbutils/DbUpdater.java

```diff
@@ -114,6 +114,9 @@ public void updateDb() {
             // Выполняем итерацию по гранулам.
             boolean success = true;
             for (Grain g : grains) {
+                if (! g.isAutoupdate()) {
```

Comment:
1. Removing `if (! g.isAutoupdate()) continue` on line 117 would mean we'd get grains **with no autoupdate** be inserted into DB since branch
```
if (gi == null) {
    insertGrainRec(g);
    success = updateGrain(g, connectionPool) & success;
}
```
would take place. Is that what we need?

2. lock flag was introduced on purpose in order to add test case 2. in [#107](https://github.com/CourseOrchestra/celesta/issues/107) but I think I need to set lock status over ISchemaCursor, so I'll remove the lock flag.